servers/lib/script/test.bash

```diff
@@ -12,20 +12,27 @@ export TEST_PATH="$test_path"
 mode=$1
 
 if [ "$mode" == "-a" ]; then
-  printf "making all tests ... and retrieving coverage report"
-  jest --coverage
+	printf "making all tests ... and retrieving coverage report"
+	jest --coverage
 elif [ "$mode" == "-u" ]; then
-  printf "making unit tests"
-  jest ../test/unit
+	printf "making unit tests"
+	jest ../test/unit
 elif [ "$mode" == "-i" ]; then
-  printf "making integration tests"
-  jest ../test/integration
+	printf "making integration tests"
+	jest ../test/integration
 elif [ "$mode" == "-e" ]; then
-  printf "making e2e tests"
-  jest --config ./test/jest-e2e.json
+	printf "making e2e tests"
+	jest --config ./test/jest-e2e.json
+elif [ "$mode" == "-c" ]; then
+	printf "starting cloudcmd instance\n"
+	libms -c .env -f ./config/cloudcmd.json >/dev/null &
+	printf "making cloudcmd tests\n"
+	jest ../test/cloudcmd
+	printf "Closing ALL cloudcmd instance...\n"
+	kill $(ps aux | grep '[l]ibms' | awk '{print $2}')
```

Comment:
you can use npx kill-port .... to achieve the same result. A fragment from [client/package.json](https://github.com/INTO-CPS-Association/DTaaS/blob/feature/distributed-demo/client/package.json).:

```sh
"test:e2e": "yarn build && yarn configapp:test && npx kill-port 4000 && yarn start >/dev/null & playwright test && npx kill-port 4000",
```

Reply:
I tried using the `npx kill-port` which did not work for me at least. It didn't shut down the server after testing. 
That is why I used this command instead, which doesn't require any additional packages.